dbhydra/dbhydra_core.py

```diff
@@ -763,7 +770,7 @@ def update(self, variable_assign, where=None):
         else:
             query = "UPDATE " + self.name + " SET " + variable_assign + " WHERE " + where
         print(query)
-        self.db1.execute(query)
+        return self.db1.execute(query)
```

Comment:
it is not clear to me what the returned value should be?

Reply:
For some SQL DBs, when a statement is executed successfully, DB can return some information. E.g. for INSERT, DB will return a PK id under which the record was inserted.